File: kyw0716/4주차/최댓값과 최솟값.js
function solution(s) {
  return `${Math.min(...s.split(" ").map((v) => Number(v)))} ${Math.max(
    ...s.split(" ").map((v) => Number(v))
  )}`;
}
// https://school.programmers.co.kr/learn/courses/30/lessons/12939

File: sk000801/LEVEL2/최댓값과 최솟값.java
class Solution {
    public String solution(String s) {
        String[] string  = s.split(" ");
        int[] string2 = new int[string.length];
        for(int i=0; i<string.length; i++) {
            string2[i] = Integer.parseInt(string[i]);
        }
        int max = string2[0];
        int min = string2[0];
        for(int i=1; i<string2.length; i++) {
            max = Math.max(max, string2[i]);
            min = Math.min(min, string2[i]);
        }
        String answer = Integer.toString(min) + " " + Integer.toString(max);
        return answer;
    }
}